content/en/docs/How To/create_output.md

```diff
@@ -0,0 +1,122 @@
+[OutputRecord](https://pkg.go.dev/github.com/AdRoll/baker#OutputRecord) objects and a channel to
+communicate to the Upload components what to upload.
+
+`CanShard` is the function telling whether the output is able to manage sharding. Read the page
+[dedicated to the sharding](/docs/how-to/sharding/) to go deeper in the topic.
+
+`Stats` is used to collect metrics, see the [dedicated page](/docs/how-to/metrics/).
+
+## OutputDesc
+
+```go
+var MyOutputDesc = baker.OutputDesc{
+	Name:   "MyOutput",
+	New:    NewMyOutput,
+    Config: &MyOutputConfig{},
+    Raw:    true,
+	Help:   "Help message for the output",
+}
+```
+
+This object has a `Name`, that is used in the Baker configuration file to identify the output,
+a constructor-like function (`New`), a config object (used to parse the output configuration in the
+TOML file) and a help text that must help the users to use the component and its configuration
+parameters. The `Raw` field tells whether the output expects to receive raw records in addition
+to single fields (see below for details).
+
+### Output constructor-like function
+
+The `New` key in the `OutputDesc` object represents the constructor-like function.
+
+The function receives an [OutputParams](https://pkg.go.dev/github.com/AdRoll/baker#OutputParams)
+object and returns an instance of [Output](https://pkg.go.dev/github.com/AdRoll/baker#Output).
+
+`OutputParams` contains the index of the output and the fields `FieldIndex` that the output will
+receive during processing, in addition to the fields inherited from
+[ComponentParams](https://pkg.go.dev/github.com/AdRoll/baker#ComponentParams).
+
+`OutputParams.Index` indicates a unique index of the output process among the concurrent output
+processes generated by Baker. The `procs` configuration can be used to tune the total number of
+concurrent processes, see [Pipeline configuration](/docs/how-to/pipeline_configuration/) for details.  
+Note that the output should extensively document in `OutputDesc.Help` if it is able to manage
+concurrent processing or the user should configure it with a single process (`procs=1`).
+Read [Tuning concurrency](/docs/how-to/concurrency/) for an in-depth study on the subject.
+
+`OutputParams.Fields` is a list of [FieldIndex](https://pkg.go.dev/github.com/AdRoll/baker#FieldIndex)
+that the output will receive. Their position in this slice correspond to the fields in
+[OutputRecord.Fields](https://pkg.go.dev/github.com/AdRoll/baker#OutputRecord), see below for
+details.  
+If, for any reason, the output needs to retrieve the fields name (like the SQLite output does to
+get the columns names), then `OutputParams.FieldName` can be used.
+
+### The output configuration and help
+
+The output configuration object (`MyOutputConfig` in the previous example) must export all
+configuration parameters that the user can set in the TOML topology file.
+
+Each key in the struct must include a `help` string tag and a `required` boolean tag, both are
```

Comment:
key/field

`required` is not required though, it defaults to `false`.

Reply:
there's also the default `tag` but as we said, we should just link to a section (that doesn't exist yet) which details configuration structs requirements and capabilities.

Reply:
fixed